bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/resources/OH-INF/thing/daikinmadoka.xml

```diff
@@ -129,4 +139,28 @@
 		</command>
 	</channel-type>
 
+	<channel-type id="brc1h_eyeBrightness">
+		<item-type>Number</item-type>
+		<label>Eye Illumination Brightness</label>
+		<state min="0" max="19" step="1" readOnly="false"/>
```

Comment:
Perhaps you could make this a Dimmer instead?

Reply:
Done - with a % conversion.